examples/rapids/mlflow_project/README.md

```diff
@@ -0,0 +1,39 @@
+### Train and Publish Locally With RAPIDS and MLFlow
+**[RAPIDS](https://rapids.ai/)** is a suite of open source libraries for GPU-accelerated analytics. 
+
+**[RAPIDS cuML](https://github.com/rapidsai/cuml)** matches the scikit-learn API, so it can build on MLFlow's existing support for scikit-learn-like models to support 
+persistence and deployment."
+
+#### Jupyter Notebook Workflow
+[Jupyter Notebook](notebooks/rapids_mlflow.ipynb)
+
+#### CLI Based Workflow
+1. Create a new conda environment.
+    1. `$ conda create -f envs/conda.yaml`
```

Comment:
Shouldn't `mlflow run` create this conda environment for the user? We might be able to remove this step